src/TorchSharp/TorchSharp.csproj

```diff
@@ -3,19 +3,24 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
-      <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+      <TargetFrameworks>netcoreapp3.1;netstandard2.0</TargetFrameworks>
       <IncludeInPackage>TorchSharp</IncludeInPackage>
       <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
       <UseMLCodeAnalyzer>false</UseMLCodeAnalyzer>
       <UseStyleCopAnalyzer>false</UseStyleCopAnalyzer>
       <IsPackable>false</IsPackable>
       <DefineConstants>$(DefineConstants);LIBTORCH_$(LibTorchPackageVersion.Replace('.', '_'));CUDA_$(CudaVersionDot.Replace('.', '_'))</DefineConstants>
+      <DefineConstants Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == 'netstandard2.0'">$(DefineConstants);NETSTANDARD2_0</DefineConstants>
```

Comment:
Same comment here, you don't need to define this, the SDK already does this for you. The one to use in this case is named: `NETSTANDARD2_0_OR_GREATER`